orchestrator/api/api_v1/endpoints/subscriptions.py

```diff
@@ -83,14 +83,19 @@ def subscription_details_by_id_with_domain_model(subscription_id: UUID) -> Dict[
 
     subs_obj = SubscriptionModel.from_subscription(subscription_id)
     subscription = subs_obj.dict()
+    sub_instance_ids = [subs_obj.subscription_id]
```

Comment:
I wonder why the old code checked if in_use_by_id was equal to subscription_id. The relation is between SubscriptionInstances and not Subscriptions, so I think the subscription_id can be omitted (in a future change)